pages/api/projects/[id]/index.js

```diff
@@ -16,4 +16,15 @@ export default async function handler(request, response) {
       return response.status(404).json({ status: "Page not found" });
     }
   }
+
+  if (request.method === "DELETE") {
+    try {
+      await Project.findByIdAndDelete(id);
+    } catch (error) {
+      console.error(`Can't delete project ${id}: ${error}`);
+    }
+
+    return response.status(201).json();
```

Comment:
```suggestion
    return response.status(200).json({ status: "Project deleted successfully" });
```
The status 201 is typically used to indicate the successful creation of a new resource, not the deletion of an existing one. Therefore, it's not commonly used for DELETE requests, where status codes like 204 (No Content) or 200 (OK) are more appropriate.